### DISCUSSION TITLE
Different version deployed to custom domain

### DISCUSSION THREAD

**Discussion by hungfhp** [vercel/community#2411] [Help]
### Summary

Deployed web is not the same version on 3 domains:

### Additional information

After deployed to vercel with Nextjs, my web is not the same version on 3 domains:

[www.bringhacks.com](http://www.bringhacks.com/)
[bringhacks.com](http://bringhacks.com/)
[bh-web-blog.vercel.app](https://bh-web-blog.vercel.app/)

Why it happen? How can fix it?

<img width="1013" alt="Screenshot 2023-05-11 at 14 04 02" src="https://github.com/vercel/community/assets/18446429/75887f6a-fed0-471c-a0a2-8d47d62b92f3">

**Comment by amyegan** [comment]
Hi @hungfhp! It looks like the first two domains may need a little more time for DNS records to propagate. The [DNS troubleshooting guide](https://vercel.com/guides/how-can-i-debug-a-dns-record) shows how you can check the records. You can also use tools like [whatsmydns.net](https://www.whatsmydns.net/#A/bringhacks.com) to check global propagation.

**Comment by hungfhp** [comment]
@amyegan 
Thank you.
I followed thi guide: https://vercel.com/guides/how-can-i-debug-a-dns-record#apex-domain-and-a-records

After run `dig vercel.com +short`, I see 2 dns:
1. 76.76.21.123
2. 76.76.21.241

The guide said `If your A Record is configured correctly, you will have a single IP address appear (76.76.21.21). If you see nothing or more than one IP, double check your DNS record to make sure it is set correctly.`

But I don't know how to `double check your DNS record to make sure it is set correctly.`
Can you give me some ideas?

**Comment by hungfhp** [comment]
One more problem is different content is rendering, 2 different content when I go to [/posts](https://bringhacks.com/posts) page in 2 different ways: 

1. Go direct to https://bringhacks.com/posts
2. Go to https://bringhacks.com/tags -> Click to header "Life Hacks" to redirect to https://bringhacks.com/posts

The content of https://bringhacks.com/posts is different on 1 & 2.
